internal/oauth/oauth.go

```diff
@@ -15,24 +15,31 @@
 package oauth
 
 import (
+	"fmt"
 	"net"
 	"net/http"
+	"os"
+	"strings"
 	"time"
 
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
 	"go.mongodb.org/atlas/auth"
 )
 
 const (
-	timeout               = 5 * time.Second
-	keepAlive             = 30 * time.Second
-	maxIdleConns          = 5
-	maxIdleConnsPerHost   = 4
-	idleConnTimeout       = 30 * time.Second
-	expectContinueTimeout = 1 * time.Second
-	cloudGovServiceURL    = "https://cloud.mongodbgov.com/"
+	timeout                    = 5 * time.Second
+	keepAlive                  = 30 * time.Second
+	maxIdleConns               = 5
+	maxIdleConnsPerHost        = 4
+	idleConnTimeout            = 30 * time.Second
+	expectContinueTimeout      = 1 * time.Second
+	cloudGovServiceURL         = "https://cloud.mongodbgov.com/"
+	userAgentContainerHostName = "container"
+	userAgentNativeHostName    = "native"
```

Comment:
Q: Can we set "native" by default in [profile.go](https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/config/profile.go#L76)?

Then conditionally replace it in this file?

Reply:
Seems reasonable I changed the approach